components/Panel/Panel.tsx

```diff
@@ -38,22 +38,19 @@ export function Panel() {
   const isMenu = panelType === "menu";
   const closeButtonColor = isMenu ? black : white;
 
-  function closePanel() {
-    setPanelOpen(false);
-  }
   return (
     <>
       {transitions(
         ({ opacity, right }, isOpen) =>
           isOpen && (
             <Overlay
-              onDismiss={closePanel}
+              onDismiss={() => closePanel(true)}
```

Comment:
This is where we use the `clearPreviousPanelData` arg — we want to let the user close everything when they click away.